src/cleanvision/issue_managers/image_property_issue_manager.py

```diff
@@ -66,7 +65,11 @@ def get_default_params(self) -> Dict[str, Any]:
                 "threshold": 0.3,
                 "normalizing_factor": 0.1,
             },
-            IssueType.BLURRY.value: {"threshold": 0.17, "normalizing_factor": 0.01},
+            IssueType.BLURRY.value: {
+                "threshold": 0.29,
+                "normalizing_factor": 0.01,
+                "color_threshold": 0.18,
```

Comment:
No need to do it in code, just here in the discussion:

Why is the threshold bumped, but the color_threshold is similar to the old threshold?

Reply:
Updated description